app/templates/base.html

```diff
@@ -88,6 +92,7 @@
                   <li><a href="{{ url_for('view_food_resource_types') }}">Manage Food Resource Types</a></li>
                   <li><a href="{{ url_for('new_food_resource_type') }}">Add Food Resource Type</a></li>
                   <li><a href="{{ url_for('user.register') }}">Invite Admins</a></li>
+                  <li><a href="{{ url_for('files') }}">Bulk Import and Output</a></li>
```

Comment:
What do you think about asking our clients about what they think of the name "Bulk Import and Output"? I'm concerned that it may not be as self-explanatory as we'd like.